cmd/skaffold/skaffold.go

```diff
@@ -19,20 +19,38 @@ package main
 import (
 	"context"
 	"errors"
+	"log"
 	"os"
 
+	"cloud.google.com/go/profiler"
 	"github.com/sirupsen/logrus"
 
 	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/instrumentation"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/output"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/version"
 )
 
 type ExitCoder interface {
 	ExitCode() int
 }
 
 func main() {
+	if _, ok := os.LookupEnv("SKAFFOLD_PROFILE"); ok {
+		err := profiler.Start(profiler.Config{
+			Service:              "skaffold_examples_microservices",
```

Comment:
What does is service mean here? Sorry I don't have context on `cloud.google.com/go/profiler`?
Shd this service be deployed in the project?

Reply:
Changed this to be configurable via env var - SKAFFOLD_PROFILER_SERVICE